openstack.rc

```diff
@@ -0,0 +1,12 @@
+export OS_REGION_NAME=RegionOne
```

Comment:
Out of curiosity, what is the source of this configuration? I.e., how do you know it is RegionOne? Is this for the swift integration tests?

Reply:
I think this file is checked in accidentally.

Reply:
yes, I'll delete it right away.